packages/block-library/src/social-links/index.js

```diff
@@ -15,10 +15,11 @@ const { name } = metadata;
 export { metadata, name };
 
 export const settings = {
-	title: __( 'Social links' ),
+	title: __( 'Social Icons' ),
 	description: __(
 		'Create a block of links to your social media or external sites'
```

Comment:
Should we update `sites` to `services` as well?:
```suggestion
		'Create a block of links to your social media or external services'
```

Reply:
In this case, because it’s a user-facing message, I’m thinking that “site” may be clearer than the more abstract “services”. No?

I agree about the full stop, good spot.